extensions/amp-video/0.1/amp-video.js

```diff
@@ -622,13 +628,58 @@ class AmpVideo extends AMP.BaseElement {
     return ranges;
   }
 
+  /**
+   * Called when video is first loaded.
+   * @override
+   */
+  firstLayoutCompleted() {
+    if (this.getPlaceholder() && !this.hideBlurryPlaceholder_()) {
```

Comment:
Let's remove `this.getPlaceholder()`. My old comment is obsolete now that it's moved outside of `hideBlurryPlaceholder_` and I looked at `togglePlaceholder` and it also does this check.